src/web/templates/admin/accounts/account_modal.html

```diff
@@ -35,10 +35,19 @@ <h4>
                     </h4>
                     <div class="row mb-3">
                         {{ admin_macros.text_input(
-                            name='username',
-                            label=_('ID (unique):'),
-                            placeholder=_('E.g.: john'),
-                            help_text=_('The Unique ID, used to authenticate.'),
+                            name='last_name',
+                            label=_('Last name:'),
+                            placeholder=_('E.g.: DOE'),
+                            help_text=_('The first name.'),
+                            data=data, errors=errors
+                        ) }}
+                    </div>
+                    <div class="row mb-3">
+                        {{ admin_macros.text_input(
+                            name='first_name',
+                            label=_('First name:'),
+                            placeholder=_('E.g.: John'),
+                            help_text=_('The last name.'),
```

Comment:
```suggestion
                            name='last_name',
                            label=_('Last name:'),
                            placeholder=_('E.g.: DOE'),
                            help_text='',
                            data=data, errors=errors
                        ) }}
                    </div>
                    <div class="row mb-3">
                        {{ admin_macros.text_input(
                            name='first_name',
                            label=_('First name:'),
                            placeholder=_('E.g.: John'),
                            help_text='',
```
Funny swap ^^  Besides that, the helptexts is useless. If you look at all the recent forms (prizes, rotators, display controllers) none of them have helptexts, or only when it is necessary. When the form is in one column it also does not make the layout feel weird to not have a help text so please try to only add some if necessary